ubuntu22.04/Dockerfile

```diff
@@ -80,16 +80,22 @@ RUN if [ "$DRIVER_TYPE" != "vgpu" ]; then \
     cd drivers && \
     /tmp/install.sh download_installer; fi
 
-# Fabric manager packages are not available for arm64
-RUN if [ "$DRIVER_TYPE" != "vgpu" ] && [ "$TARGETARCH" != "arm64" ]; then \
+RUN if [ "$DRIVER_TYPE" != "vgpu" ]; then \
```

Comment:
nit: not a blocker, but I would have preferred if this change was a separate commit.

Reply:
fair enough. I can raise a separate PR for that

Reply:
Done, please check - https://github.com/NVIDIA/gpu-driver-container/pull/278